instrumentation/cassandra/cassandra-4.4/testing/src/main/java/io/opentelemetry/testing/cassandra/v4_4/AbstractCassandra44Test.java

```diff
@@ -135,4 +144,23 @@ private static Stream<Arguments> provideReactiveParameters() {
                     "SELECT",
                     "users"))));
   }
+
+  @Override
+  protected CqlSession getSession(String keyspace) {
+    DriverConfigLoader configLoader =
+        DriverConfigLoader.programmaticBuilder()
+            .withDuration(DefaultDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(0))
+            .withDuration(DefaultDriverOption.CONNECTION_INIT_QUERY_TIMEOUT, Duration.ofSeconds(10))
+            .build();
+    return wrap(
+        CqlSession.builder()
+            .addContactEndPoint(
+                new SniEndPoint(
+                    new InetSocketAddress("localhost", cassandraPort),
+                    "localhost:" + cassandraPort))
```

Comment:
can you pass in anything as `serverName`? just wondering if my assumption that `serverName` represents `{server.address}:{server.port}` is not correct

Reply:
GenericContainer.getContainerName.. there is no such thing as server.address

Reply:
from cassandra-java-driver repo

https://github.com/search?q=repo%3Aapache%2Fcassandra-java-driver%20new%20SniEndPoint&type=code

`return new SniEndPoint(cloudProxyAddress, hostId.toString());`  // it's using hostId.. there is no port

Reply:
one option is to access proxyAddress of SniEndpoint to get server.address and server.port.. but that is private final..

Reply:
@laurit thoughts on this?